src/preprocessing/passes/bv_to_int.h

```diff
@@ -39,8 +39,8 @@ class BVToInt : public PreprocessingPass
   PreprocessingPassResult applyInternal(
       AssertionPipeline* assertionsToPreprocess) override;
 
-  void addFinalizeRangeAssertions(AssertionPipeline* assertionsToPreprocess,
-                                  std::vector<Node> rangeConstraints);
+  void addFinalizeAssertions(AssertionPipeline* assertionsToPreprocess,
+                             std::vector<Node> additionalConstraints);
```

Comment:
Although not introduced on this PR, this should probably not be a vector value, either `std::vector<Node>&` or `const std::vector<Node>&` to avoid copying.

Reply:
Good catch! Addressed with 004e3a61b5c64c2fb339f49b1a3c5e572a7dc1f8.